vmdb/app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -492,16 +492,11 @@ def build_template_grid(templates,sort_order="DESC",sort_by="name")
   end
 
   def dialog_partial_for_workflow
-    case (@edit || @options).try(:[], :wf)
-    when MiqProvisionVirtWorkflow
-      "prov_dialog"
-    when MiqProvisionConfiguredSystemForemanWorkflow
-      "prov_configured_system_foreman_dialog"
-    when MiqHostProvisionWorkflow
-      "prov_host_dialog"
-    when VmMigrateWorkflow
-      "prov_vm_migrate_dialog"
-    end
+    workflow = (@edit || @options).try(:[], :wf)
+    return "prov_dialog"                           if workflow.kind_of?(MiqProvisionVirtWorkflow)
+    return "prov_configured_system_foreman_dialog" if workflow.kind_of?(MiqProvisionConfiguredSystemForemanWorkflow)
+    return "prov_host_dialog"                      if workflow.kind_of?(MiqHostProvisionWorkflow)
+    return "prov_vm_migrate_dialog"                if workflow.kind_of?(VmMigrateWorkflow)
```

Comment:
@brandondunne i think the issue was that the code couldn't find prov_dialog partial to render in catalog or vm_infra directories we have to return miq_request/prov_dialog